share_extension/android/extension_post/extension_post.js

```diff
@@ -312,13 +313,13 @@ export default class ExtensionPost extends PureComponent {
 
             const value = text.join('\n');
 
-            if (!error) {
+            if (!error && files.length <= 5 && totalSize <= maxFileSize) {
```

Comment:
We may use the constant for max file count,
`import {MAX_FILE_COUNT} from 'app/constants/post_textbox';`

Reply:
I will move that constant from there to avoid cycling requires